cpp/benchmarks/reduction/anyall.cpp

```diff
@@ -34,14 +36,18 @@ void BM_reduction_anyall(benchmark::State& state,
   auto const dtype           = cudf::type_to_id<type>();
   data_profile const profile = data_profile_builder().no_validity().distribution(
     dtype, distribution_id::UNIFORM, 0, agg->kind == cudf::aggregation::ANY ? 0 : 100);
-  auto const values = create_random_column(dtype, row_count{column_size}, profile);
+  auto values = create_random_column(dtype, row_count{column_size}, profile);
 
   cudf::data_type output_dtype{cudf::type_id::BOOL8};
 
   for (auto _ : state) {
     cuda_event_timer timer(state, true);
     auto result = cudf::reduce(*values, *agg, output_dtype);
   }
+
+  // The benchmark takes a column and produces one scalar.
+  set_items_processed(state, column_size + 1);
+  set_bytes_processed(state, estimate_size(std::move(values)) + cudf::size_of(output_dtype));
```

Comment:
```suggestion
  set_bytes_processed(state, estimate_size(*values) + cudf::size_of(output_dtype));
```
similarly at other places.

Reply:
I think `values->view()` is more clear but leave it up to you if you'd rather use `*values`

Reply:
Hi @karthikeyann, thanks for the review. I just want to better understand your comment. Your seem to be suggesting to pass a `column_view` instead of moving the `column`. If this is your suggestion, it has been done in https://github.com/rapidsai/cudf/pull/16126/commits/40804e27cd510f746e92105a23e7feb9a35d6cbe. Or, are you suggesting to use the `*`?